core/src/main/kotlin/com/ritense/valtimo/importchangelog/ImportChangelogDeploymentService.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ *  Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.ritense.valtimo.importchangelog
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.ritense.valtimo.contract.importchangelog.ChangesetDeployer
+import com.ritense.valtimo.importchangelog.domain.ImportChangeset
+import com.ritense.valtimo.importchangelog.repository.ImportChangesetRepository
+import liquibase.util.MD5Util
+import liquibase.util.StringUtil
+import mu.KotlinLogging
+import org.springframework.boot.context.event.ApplicationReadyEvent
+import org.springframework.context.event.EventListener
+import org.springframework.core.io.Resource
+import org.springframework.core.io.ResourceLoader
+import org.springframework.core.io.support.ResourcePatternUtils
+import org.springframework.transaction.annotation.Transactional
+import java.io.IOException
+import java.text.Normalizer
+import java.time.Instant
+import kotlin.jvm.optionals.getOrNull
+
+@Transactional
+class ImportChangelogDeploymentService(
```

Comment:
Thinking about the future: Let's say we wanted to do this for Form Flow as well, how would we do this without adding a dependency on core to the module?

Reply:
By using the `contract` module. I've done the same in this PR for the `authorization` module. Because the `authorization` module also doesn't have a dependency on `core`

But you could also add `core` to `form-flow-valtimo`